pkg/nodeconfig/nodeconfig.go

```diff
@@ -4,10 +4,13 @@ import (
 	"context"
 	"crypto/sha256"
 	"fmt"
+	"net"
 	"net/url"
 	"strings"
 	"time"
 
+	"github.com/apparentlymart/go-cidr/cidr"
+
```

Comment:
Unnecessary new line. PS: The formatter should fix this for you.

Reply:
Addressed, `gofmt` did not caught that.